src/pages/community/index.js

```diff
@@ -9,21 +9,34 @@ import Footer from "../../sections/General/Footer";
 import Community from "../../sections/Community";
 
 import { GlobalStyle } from "../../sections/app.style";
-import theme from "../../theme/app/themeStyles";
-
-const CommunityIndex = () => (
-  <ThemeProvider theme={theme}>
-    <Layout>
-      <GlobalStyle />
-      <SEO title="Cloud Native, Kubernetes, and Service Mesh Community"
-        description="The Cloud Native Management Community at Layer5 - Learn, Share, Engage.
+import lighttheme from "../../theme/app/themeStyles";
+import darktheme from "../../theme/app/darkThemeStyles";
+import { useState } from "react";
+import { useCookies } from "react-cookie";
+import { useEffect } from "react";
+const CommunityIndex = () => {
+  const [cookies, setCookie] = useCookies(["user"]);
+  const [theme, setTheme] = useState("light");
+  useEffect(() => {
+    if(cookies.Theme !== undefined)
+      setTheme(cookies.Theme);
+  }, []);
+  const themeSetter = (thememode) => {
+    setTheme(thememode);
+  };
```

Comment:
the `themeSetter` is almost used in all the pages. Export this function instead that can be used globally, without defining it again.

Reply:
Ok👍